apps/studio/src/features/editing-experience/components/CreateFolderModal/CreateFolderModal.tsx

```diff
@@ -146,6 +147,7 @@ const CreateFolderModalContent = ({
               </FormLabel>
               <Input
                 placeholder="This is a url for your new page"
+                maxLength={MAX_FOLDER_PERMALINK_LENGTH}
```

Comment:
dont need to. this is because `generateResourceUrl` is derived from the `title`, which is constrained to a shorter length than the `permalink`. 

in addition, we only do this generation when the field `isClean` (ie, the user has not edited it). hence, once the user clicks and makes an edit, we no longer generate the url automatically. hence, the field is always constrained to <= `maxLength`

Reply:
Hmm, I think we might need to add a comment here or something, because I thought it wasn't obvious that because title's length is always shorter than the permalink, we will not need to check for that case.

Other than that no other comments.

Reply:
sounds good, lemme add that in